orchagent/orchdaemon.cpp

```diff
@@ -665,45 +665,28 @@ bool OrchDaemon::warmRestartCheck()
 }
 
 void
-OrchDaemon::check_and_exit()
+OrchDaemon::checkAndExit()
 {
     SWSS_LOG_ENTER();
 
-    swss::SubscriberStateTable tbl(m_configDb, "CONTAINER_FEATURE");
-    std::deque<KeyOpFieldsValuesTuple> entries;
+    swss::Table tbl(m_configDb, "CONTAINER_FEATURE");
+    string val;
 
-    tbl.pops(entries);
-
-    for (auto& entry: entries) {
-        string key = kfvKey(entry);
-
-        if (key == "swss") {
-            auto values = kfvFieldsValues(entry);
-
-            for (auto it : values)
-            {
-                string field = fvField(it);
-                string value = fvValue(it);
-
-                if (field == "orchagent_exit_on_switch_shutdown") {
-                    if (value == "enable") {
-                        SWSS_LOG_ERROR("Exiting....");
-                        exit(EXIT_FAILURE);
-                    }
-                }
-            }
-            break;
+    if (tbl.hget("swss", "orchagent_exit_on_switch_shutdown", val)) {
```

Comment:
@renukamanavalan , `{ `to new line :). Same as below

Reply:
Thanks. In my last places, the style requirement is to place the brace in the same line. Hard to shake that. :-)

Reply:
done.